lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM.cpp

```diff
@@ -576,7 +576,8 @@ struct ConvertLayoutOpConversion
     unsigned inVec = 0;
     unsigned outVec = 0;
     auto paddedRepShape = getScratchConfigForCvtLayout(op, inVec, outVec);
-    if (getElementTypeOrSelf(op.getType()).isa<mlir::Float8E4M3B11FNUZType>()) {
+    if (getElementTypeOrSelf(op.getType()).isa<mlir::Float8E4M3B11FNUZType>() ||
+        getElementTypeOrSelf(op.getType()).isa<mlir::Float8E4M3FNType>()) {
```

Comment:
nit trick you can write this :)
`getElementTypeOrSelf(op.getType()).isa<mlir::Float8E4M3B11FNUZType, mlir::Float8E4M3FNType>()`

Reply:
oh nice!